backend/src/test/resources/BasicWorldTest.json

```diff
@@ -5,7 +5,7 @@
   "regions": [{
     "kingdom": "k1",
     "type": "land",
-    "relgion": "Sword of Truth",
+    "religion": "Sword of Truth",
```

Comment:
On reflection, should the value be "Iruhan (Sword of Truth)"?
Or, can the line be dropped altogether?

Reply:
For the object to create properly it should, however this class and the associated JSON files are no longer needed and are not used anymore by any tests.  I should have just deleted all this stuff.